src/selection.js

```diff
@@ -39,8 +39,8 @@ export function getEndLineRect(range, isForward) {
     }
 
     return {
-        top: Math.min(...endLineRects.map(rect => rect.top)),
-        bottom: Math.max(...endLineRects.map(rect => rect.bottom)),
+        top: window.pageYOffset + Math.min(...endLineRects.map(rect => rect.top)),
+        bottom: window.pageYOffset + Math.max(...endLineRects.map(rect => rect.bottom)),
```

Comment:
I see two minor problems here:
1. `window` is a reference to the main window object, which might not be what we want (e.g. in a `<iframe>`);
2. the name `getEndLineRect` recalls other methods like `getClientRects` or `getBoundingClientRect`, where the "rects" are `ClientRect` objects, with their `top`, `right`, `bottom` and `left` properties being distances from the *viewport*'s borders: that's why this function doesn't add scrolling.

I think a better place to add vertical scrolling would be `stylePopover` in popover.js, where the style is actually applied. It also uses `getPageScroll` from utils.js which should provide a cross-browser way to retrieve the page scrolling (not that I care about IE8- anyway).